src/services/auth.service.ts

```diff
@@ -152,34 +186,36 @@ export async function handleVerifyLoginDeviceOtp(body: { otp: string, email: str
   }
 
   /**send a login token to the user*/
-  const accessToken = jwtHelper.generateToken(
+  const accessToken = jwtHelper.generateToken({
+    email,
+    deviceId,
+    type: JwtType.USER,
+    userId: existingUserAuth?.user,
+  });
+
+  /**for first time login -> upsert-true*/
+  await UserTokenDb.updateOne(
+    { email, user: existingUserAuth?.user },
     {
       email,
+      token: accessToken,
+      user: existingUserAuth?.user,
       deviceId,
-      type: JwtType.USER,
-      userId: existingUserAuth?.user
-    });
-
-  /**for first time login -> upsert-true*/
-  await UserTokenDb.updateOne({ email, user: existingUserAuth?.user }, {
-    email,
-    token: accessToken,
-    user: existingUserAuth?.user,
-    deviceId
-  }, { upsert: true });
+    },
+    { upsert: true },
+  );
 
   return accessToken;
 }
 
-export async function googleAuth(body: { email: string, googleToken: string, deviceId: string }): Promise<string> {
-
+export async function googleAuth(body: { email: string; googleToken: string; deviceId: string }): Promise<string> {
   const { googleToken, deviceId } = body;
   /**pull it off separately, so I can change it to lowercase */
   const email = body.email.toLowerCase();
 
   const existingUserAuth = await UserAuthDb.findOne({ email });
 
-  if (existingUserAuth && (existingUserAuth.type === AuthType.EMAIL)) {
+  if (existingUserAuth && existingUserAuth.type === AuthType.EMAIL) {
```

Comment:
Return the parenthesis, they are there for a reason. it's to make code more readable and avoid confusion in the future.